CONTRIBUTING.md

```diff
@@ -26,11 +26,12 @@ See [sdk-structure.md](./docs/sdk-structure.md)
   git submodule init
   git submodule update
   ```
+  > if you get a `The authenticity of host 'github.com (192.30.252.123)' can't be established.` error, run `ssh-keyscan github.com >> ~/.ssh/known_hosts` and retry.
```

Comment:
If

Reply:
I've been using this for over a decade, nice to know about `ssh-keyscan` (in an alias)
```
perl -i -n -e "print unless (\$. == $1)" ~/.ssh/known_hosts <line_number>
```

Reply:
```suggestion
  > If you get a `The authenticity of host 'github.com (192.30.252.123)' can't be established.` error, run `ssh-keyscan github.com >> ~/.ssh/known_hosts` and retry.
```